src/platform/lumin-runtime/elements/builders/progress-bar-builder.js

```diff
@@ -53,7 +53,7 @@ export class ProgressBarBuilder extends UiNodeBuilder {
     setProgressColor (element, oldProperties, newProperties) {
         const { beginColor, endColor } = newProperties.progressColor;
         if ( beginColor !== undefined && endColor !== undefined ) {
-            element.setProgressColor( beginColor, endColor );
+            element.setProgressColor( ColorProperty.validate(beginColor), ColorProperty.validate(endColor) );
```

Comment:
Instead of `validate` it should be `parse`:

`element.setProgressColor( ColorProperty.parse(beginColor), ColorProperty.parse(endColor) );`